CONVERTER-SERVICE/src/controllers/docFileConverter.controller.js

```diff
@@ -10,16 +10,14 @@ accordance with the terms of the license agreement you entered into
 with Jalasoft.
 */
 
-const libre = require('libreoffice-convert');
-const ConverterException = require('../Exceptions/converter.exception');
-const Upload = require('../helpers/upload.helper');
+const FileChecker = require('../helpers/fileChecker.helper');
+const ConverterChecker = require('../helpers/converterChecker.helper');
 const Converter = require('./converter.controller');
-const fs = require('fs').promises;
 
 class docFileConverterController extends Converter {
```

Comment:
Add a comment for the docFileConverterController and also you could complete the word doc to document